Resources/Public/less/bootstrap.less

```diff
@@ -868,6 +868,11 @@
 //** Horizontal line color.
 @hr-border:                   @gray-lighter;
 
+// Custom media queries breakpoints
+@screen-xs--custom: 520px;
```

Comment:
Could you explain why such numbers, just curious? (520,1090,1300)

Reply:
They were all already there. I only replaced it by variables :)

Reply:
hmm, my mistake :) But still, then better to change that css to use regular Bootstarap media queries to keep a number of variables small as much as possible.